yesod-test/Yesod/Test.hs

```diff
@@ -566,6 +570,16 @@ nameFromLabel label = do
 (<>) :: T.Text -> T.Text -> T.Text
 (<>) = T.append
 
+-- |
+-- @since 1.5.9
+byLabelWithMatch :: (T.Text -> T.Text -> Bool) -- ^ The matching method which is used to find labels (i.e. exact, contains)
```

Comment:
Is this meant to be exported? It seems really useful

Reply:
No, neither.